_content/faq.yml

```diff
@@ -27,10 +27,16 @@
       a: |-
         It depends. We strive to have you go through the interview process within four weeks. Onboarding can take an additional four to six weeks depending on when you get an offer from the HR team and how long the background check takes. Your time in the onboarding process is generally out of our control at USDS.
 
-    - q: "When will I hear from the U.S. Digital Service in this process?"
+    - q: "When will I hear from USDS in this process?"
       a: |-
         You should receive an email confirmation when your application is submitted. If you are selected to move on to interviews, you will be connected with a member of the USDS Talent Team. They will be your guide throughout your interview process.
 
+    - q: "How does USDS address diversity and inclusion?"
```

Comment:
The EEO statement alone doesn't fully address what we do regarding diversity and inclusion. Someone could read this as the _only_ thing we do is _encourage_ people to apply. This headline should either be more specifically about EEO, or we could write another graph or two about how else we support diversity and inclusion.

Reply:
Agreed, good point. I've changed the wording of the question, and asked the team who wrote this language initially to consider making a second FAQ that is more about D&I in USDS culture / day-to-day operations.

Reply:
👍